app/models/reports/schema_v2.rb

```diff
@@ -52,85 +64,27 @@ def initialize(regions, periods:)
     # Adjusted patient counts are the patient counts from three months ago (the adjusted period) that
     # are the basis for control rates. These counts DO NOT include lost to follow up.
     memoize def adjusted_patients_without_ltfu
-      regions.each_with_object({}) { |region, result|
-        result[region.slug] = adjusted_patients_without_ltfu_query(region).slice(*periods.entries)
-      }
-    end
-
-    private def adjusted_patients_without_ltfu_query(region)
-      FacilityState.for_region(region)
-        .order(:month_date)
-        .group(:month_date)
-        .sum("patients_under_care::int")
-        .to_h(&period_hash)
-        .tap { |hsh| hsh.default = 0 }
+      regions.each_with_object({}) { |region, result| result[region.slug] = sum(region, :patients_under_care) }
```

Comment:
Does it make sense to extract a `sum_by_region(regions, field)`?

Reply:
Following this trail of thought: if we're running a single query to get _all_ the data that goes into the region report on a page, then can we just return a single hash map from the repository, instead of having functions? Will that be simpler?

This doesn't have to be done now, I'm just throwing a suggestion.

Reply:
> instead of having functions? Will that be simpler?

Yeah the Repository API could change to support something like that for sure, but it would mean changing a lot of callers (or having dual interfaces).  Maybe someday down the road 😁 .  I'm ready to get to V2 and let this be mostly stable for awhile until the dashboard refresh or other features require larger changes.